srcpkgs/intel-media-driver/template

```diff
@@ -18,10 +18,49 @@ checksum=dec2210c1f824a721e4ea6f66f5899e300801b3e7fae232d3f8a94db38c2edd1
 build_options="nonfree"
 desc_option_nonfree="Enable nonfree kernels"
 
+post_configure() {
+	configure_args="${configure_args/KERNELS=OFF/KERNELS=ON}"
+	(
+		cmake_builddir="nonfree-build"
+		do_configure
+	)
+}
+
+post_build() {
+  (
+		DESTDIR=${DESTDIR/${sourcepkg}/${sourcepkg}-nonfree}
+		cmake_builddir="nonfree-build"
+		do_build
+  )
+}
+
+# for some reason, lib installs to lib64 instead of symlinking?
 post_install() {
 	vlicense LICENSE.md
+	(
+		DESTDIR=${DESTDIR/${sourcepkg}/${sourcepkg}-nonfree}
```

Comment:
better to do this in the pkg_install() function because you have `PKGDESTDIR` there

Reply:
i'm glad you said something about the build option. it seems removing this doesn't work. there is nothing in the nonfree package but empty directories (and that's why it didn't need the lib64 move)

but i also don't see nonfree build as nonfree (args are the same, total cmake steps is 1548 for both) if i use `build_options_default` instead of `configure_args`